heudiconv/tests/test_utils.py

```diff
@@ -1,10 +1,18 @@
+import json
 import os
 import os.path as op
+
+try:
+    from json.decoder import JSONDecodeError
+except ImportError:
+    JSONDecodeError = ValueError
```

Comment:
so this is to work on PY2.  I would have made it explicit.
We don't use/require `six` at runtime so doomed to use `sys.version_info[0] > 2` or define PY2, PY3 e.g. within `heudiconv.utils`.
That would help later on to find/prune python2-specific code.

Reply:
actually -- you do the same in `heudiconv.utils`, so do it there and just import here the exception